tests/core/pyspec/eth2spec/test/electra/unittests/test_config_invariants.py

```diff
@@ -8,5 +8,15 @@
 @with_electra_and_later
 @spec_test
 @single_phase
-def test_withdrawals(spec):
+def test_withdrawal_requests(spec):
     assert spec.MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD < spec.MAX_WITHDRAWALS_PER_PAYLOAD
+
+
+@with_electra_and_later
+@spec_test
+@single_phase
+def test_pending_partial_withdrawals(spec):
+    assert (
+        spec.MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD
+        == spec.MAX_PENDING_PARTIALS_PER_WITHDRAWALS_SWEEP * 2
```

Comment:
What is the rationale in keeping this invariant?

Reply:
as I understood it, this was the intention